extension/pybindings/pybindings.cpp

```diff
@@ -975,6 +979,18 @@ py::list get_operator_names() {
   return res;
 }
 
+py::list get_backend_names() {
+  size_t n_of_registered_backends = get_num_registered_backends();
+  py::list res;
```

Comment:
is the result list or set? in the typehint, it is Set[str] but you're returning a list. Which one is it?

Reply:
It should be list